### ISSUE TITLE
Support for Monterey 12.1 M1 Chip

### ISSUE THREAD

**Issue by SoumadeepChoudhury** [jwise/HoRNDIS#153]
### So that we do not need to disable SIP for installation.....

**Comment by misinglink** [comment]
yes please

**Comment by alekssze** [comment]
I would Like that also

**Comment by mrcoding-dev** [comment]
pls

**Comment by maurocos** [comment]
pls

**Comment by AlfredJKwack** [comment]
Have you tried the build and comments following this: https://github.com/jwise/HoRNDIS/issues/135#issuecomment-940140208

Or alternatively https://github.com/jwise/HoRNDIS/issues/146 also gets you there.

**Comment by ghost** [comment]
Pleaseeee

**Comment by AlfredJKwack** [comment]
@altitudedashboard see my comment above. It gets you there.

**Comment by ghost** [comment]
This workaround disables SIP which in turn on M1 chips disables the ability to run iOS apps. For some people this is not an option.

**Comment by inspiredearth** [comment]
Agreed. It would be great to have a correctly signed version of the kext which will run on M1 — WITHOUT losing the ability to run iOS apps.

**Comment by samip5** [comment]
> This workaround disables SIP which in turn on M1 chips disables the ability to run iOS apps. For some people this is not an option.

I don't think one can run iOS apps anymore on M1 macs as Apple has disabled it in MacOS 12.1 or is that just the ones that are not meant to be run on it by devs?
